packages/portal/spec-details/sandbox/App.vue

```diff
@@ -10,21 +10,39 @@
           }"
         >
           <h3>Try it Out:</h3>
-          <label>
-            <input
-              v-model="hasSidebar"
-              type="checkbox"
-            > hasSidebar
-          </label>
-          <label class="ml-3">
-            <input
-              v-model="essentialsOnly"
-              type="checkbox"
-            > essentialsOnly
-          </label>
+          <div class="props">
```

Comment:
suggestion: can we rename this class to avoid confusion with Vue native `props`?

Reply:
fixed